helpers/lib/resourceHints.js

```diff
@@ -48,11 +48,11 @@ function addResourceHint(globals, path, rel, type, cors) {
 
     let index = globals.resourceHints.findIndex(({src}) => path === src);
     if (index >= 0) {
-        return;
+        return path;
```

Comment:
There may be scenarios where nested templates include helpers that produce preload hints for the same resource where only the 1st usage of the helper will have the correct effect and the rest, will have unexpected results.

As an example let's take `stylesheet`, also, think that there is 1 `common-resources` (nested) template that is included in a `base` template. The nested template includes the `{{{ stylesheet '/asset/css/theme.css' }}}` helper which `base` template also includes but, in a linear order, the nested template is rendered before the occurrence of the `stylesheet` helper in the base template. For the nested template, the produced `link` tag will be correct, it will include a valid `href` attribute. The `stylesheet` in the `base` template:
- (with `return;`) will produce a `href="undefined"`
- (with the proposed change) will produce a valid `href` attribute, with the correct URL.